STYLE_GUIDE.md

```diff
@@ -0,0 +1,508 @@
+# Airbnb React/JSX Style Guide
```

Comment:
It looks like it completely overwrote the style guide when you pushed this branch again.We probably don't want to force push file changes, otherwise we loose our commit history.